openapi-cli/src/test/resources/ballerina-to-openapi/response/nil_union_return_type.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+
+type Links record {|
+    int id;
+    string[] links;
+|};
+
+ public type BadRequestRecord record {|
+    *http:BadRequest;
+    string body;
+ |};
+
+
+service /payloadV on new http:Listener(9090) {
+    # for `get` method with nil return type and error
+    resource function get path_get_error() returns error? {
+    };
+
+    # for `get` method with nil return type and success status code
+    resource function get path_get() returns string? {
```

Comment:
```suggestion
    resource function get path_get() returns string {
```
I guess we already have test cases for above right?

Reply:
Actually no,  we haven't this is regarding the path without any parameters that have returned with string and nil value, in line 24 we have a different scenario that returns the same with the query parameter :)